org.jacoco.cli.test/src/org/jacoco/cli/internal/commands/ReportTest.java

```diff
@@ -0,0 +1,142 @@
+/*******************************************************************************
+ * Copyright (c) 2009, 2017 Mountainminds GmbH & Co. KG and Contributors
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *    Marc R. Hoffmann - initial API and implementation
+ *
+ *******************************************************************************/
+package org.jacoco.cli.internal.commands;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.io.FileOutputStream;
+
+import org.jacoco.cli.internal.CommandTestBase;
+import org.jacoco.core.data.ExecutionData;
+import org.jacoco.core.data.ExecutionDataWriter;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+/**
+ * Unit tests for {@link Report}.
+ */
+public class ReportTest extends CommandTestBase {
+
+	@Rule
+	public TemporaryFolder tmp = new TemporaryFolder();
+
+	@Test
+	public void should_print_usage_when_no_options_are_given()
+			throws Exception {
+		execute("report");
+
+		assertFailure();
+		assertContains("Option \"--classfiles\" is required", err);
+		assertContains(
+				"Usage: java -jar jacococli.jar report [<execfiles> ...]", err);
+	}
+
+	@Test
+	public void should_print_number_of_analyzed_classes() throws Exception {
+		execute("report", "--classfiles", getClassPath());
+
+		assertOk();
+		assertContains("[INFO] Writing report with 14 classes.", out);
```

Comment:
@marchof as discussed in https://github.com/jacoco/jacoco/pull/525#discussion_r117818267 - `Analyzing 14 classes.` sounds better